api.php

```diff
@@ -377,6 +377,71 @@ public function run($userID, $permissionIsExplicit) {
 	}
 }
 
+// FIXME - a bit copypasta with the above API call togglevote.
+// togglevote also uses GET rather than POST, but GET is not supposed to be used
+// for state-modifying web queries. So probably togglevote should be deprecated.
```

Comment:
My understanding is these APIs are also being used by others so it is something to keep in mind before deprecating.